### ISSUE TITLE
--frozen-lockfile breaking on github actions

### ISSUE THREAD

**Issue by shawshankkumar** [oven-sh/bun#4939]
### What version of Bun is running?

1.0.0+822a00c4d508b54f650933a73ca5f4a3af9a7983

### What platform is your computer?

Darwin 22.1.0 x86_64 i386

### What steps can reproduce the bug?

I have a small package which I am migrating to bun. I want to run bun install in the github action but it keeps failing when I use frozen lockfile. I am unable to debug this, would love some help!

https://github.com/shawshankkumar/express-status-validation/actions/runs/6146452663/job/16675853729

This is the action which is failing.

https://github.com/shawshankkumar/express-status-validation/blob/890c77ffd203256cd6fa7358fe49101234d48ade/.github/workflows/test.yaml#L31 the workflow file for this action run.

<img width="1238" alt="image" src="https://github.com/oven-sh/bun/assets/74819565/3350259c-f2d8-403c-9c79-435799e04bc1">

### What is the expected behavior?

It should run without any errors. Like it does locally.
<img width="766" alt="image" src="https://github.com/oven-sh/bun/assets/74819565/a3bcafb4-4b6d-4293-a742-af8c5cb161a3">

Note: I have pushed the lockfile, I do not understand why the error is happening.

### What do you see instead?

-

### Additional information

-

**Comment by infinitedim** [comment]
### Use --no-save instead of --frozen-lockfile

I have the same problem and after reading the official documentation of the bun I didn't find the --frozen-lockfile argument to lock the lockfile but there is a --no-save argument to install dependencies without creating a lockfile. 

```bash
bun install --no-save
```

### Generate yarn.lock file in addition bun.lockb

Another way is to add yarn.lock in addition to bun.lockb, there are two ways to add the yarn.lock file, namely with cli or with the configuration file

## with cli
```bash
bun install --yarn
```

## with config file
if you using this way you must create file called bunfig.toml and paste code down bellow

```bash
[install.lockfile]

# path to read bun.lockb from
path = "bun.lockb"

# path to save bun.lockb to
savePath = "bun.lockb"

# whether to save the lockfile to disk
save = true

# whether to save a non-Bun lockfile alongside bun.lockb
# only "yarn" is supported
print = "yarn"

```



### References
[bun official github](https://github.com/oven-sh/bun/blob/main/docs/install/lockfile.md)
[bun official docs](https://bun.sh/docs/install/lockfile)

**Comment by shawshankkumar** [comment]
@infinitedim 

Hey, thanks for replying. I'll try it out but frozen lockfile is there since v0.6
I read @Jarred-Sumner 's tweet : https://x.com/jarredsumner/status/1673554371702067200?s=20
and the docs too have it:

https://bun.sh/docs/cli/install

<img width="1792" alt="image" src="https://github.com/oven-sh/bun/assets/74819565/6800d97a-58fd-4114-ba66-6217009580f9">

**Comment by Jarred-Sumner** [comment]
We fixed some bugs in Bun v1.1.1x that would likely have covered this.

If you're still running into this, please let us know

If you want to use `--frozen-lockfile`, the lockfile will have to be up-to-date, including for things like postinstall scripts.
